partcad/src/partcad/telemetry_none.py

```
@@ -0,0 +1,74 @@
+        pass
+
+    def record_exception(self, exception, attributes=None, timestamp=None, escaped=False):
+        pass
+
+    def set_attributes(self, attributes):
+        pass
+
+    def set_attribute(self, key, value):
+        pass
+
+    def set_status(self, status):
```

Ruff (0.8.2)

Redefinition of unused `set_status` from line 18

(F811)